config/settings/base.py

```diff
@@ -56,7 +56,7 @@
 # https://docs.djangoproject.com/en/dev/ref/settings/#root-urlconf
 ROOT_URLCONF = "config.urls"
 # https://docs.djangoproject.com/en/dev/ref/settings/#wsgi-application
-WSGI_APPLICATION = "config.wsgi.application"
+# WSGI_APPLICATION = "config.wsgi.application"
```

Comment:
Does WSGI_APPLICATION need to be commented? If yes we should remove it or at least describe why it's commented out